modules/client/examples/05-encoders-decoders/12-apply-uninstallation.ts

```diff
@@ -0,0 +1,109 @@
+/* MARKDOWN
+---
+title: Apply Uninstallation
+---
+
+## Apply a Plugin Uninstallation
+
+Encodes the action of applying a plugin uninstallation.
+
+### Encoding
+*/
+
+import {
+  ApplyUninstallationParams,
+  Client,
+  DaoAction,
+  DecodedApplyUninstallationParams,
+} from "@aragon/sdk-client";
+import { hexToBytes } from "@aragon/sdk-common";
+import { context } from "../index";
+import { PermissionIds } from "../../dist/interfaces";
+
+// Instantiates an Aragon OSx SDK client.
+const client: Client = new Client(context);
+
+// This variable contains the values received on the prepareUninstallation() method
+const applyUninstallationParams: ApplyUninstallationParams = {
+  permissions: [{
+    condition: "0x1234567890123456789012345678901234567890",
```

Comment:
@Rekard0 is the condition needed to apply an uninstallation?

Reply:
Ping

Reply:
> @Rekard0 @josemarinas is the condition needed to apply an uninstallation?

yes, but it can be address zero()

Reply:
According to applyUninstallation > applyMultiTargetPermissions, the condition won't be used when revoking the permissions:

https://github.com/aragon/osx/blob/527474bb14529f2892e8277f6d7a1ca2da637a55/packages/contracts/src/core/permission/PermissionManager.sol#L173-L177